extensions/data-plane/data-plane-framework/src/test/java/org/eclipse/dataspaceconnector/dataplane/framework/manager/DataPlaneManagerImplTest.java

```diff
@@ -14,98 +14,146 @@
 package org.eclipse.dataspaceconnector.dataplane.framework.manager;
 
 import org.eclipse.dataspaceconnector.dataplane.framework.store.InMemoryDataPlaneStore;
-import org.eclipse.dataspaceconnector.dataplane.spi.pipeline.PipelineService;
+import org.eclipse.dataspaceconnector.dataplane.spi.pipeline.TransferService;
+import org.eclipse.dataspaceconnector.dataplane.spi.store.DataPlaneStore;
 import org.eclipse.dataspaceconnector.spi.monitor.Monitor;
 import org.eclipse.dataspaceconnector.spi.result.Result;
 import org.eclipse.dataspaceconnector.spi.types.domain.DataAddress;
 import org.eclipse.dataspaceconnector.spi.types.domain.transfer.DataFlowRequest;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
 
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
 
 import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.eq;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 class DataPlaneManagerImplTest {
-    private PipelineService pipelineService;
-    private DataPlaneManagerImpl dataPlaneManager;
+    private TransferService transferService = mock(TransferService.class);
+    private TransferService transferService2 = mock(TransferService.class);
+
+    DataPlaneStore store = new InMemoryDataPlaneStore(10);
+    DataFlowRequest request = createRequest();
+    CountDownLatch latch = new CountDownLatch(1);
+    ArgumentCaptor<Stream> streamCaptor = ArgumentCaptor.forClass(Stream.class);
+    TransferServiceSelectionStrategy transferServiceSelectionStrategy = mock(TransferServiceSelectionStrategy.class);
+
+    {
+        when(transferServiceSelectionStrategy.chooseTransferService(any(), any()))
+                .thenReturn(transferService);
+    }
```

Comment:
why not have this logic inside  Junit `@BeforeEach` ? This will keep it more in Junit code style and easier to read.